app/services/calculators/housing_costs_calculator.rb

```diff
@@ -5,11 +5,12 @@ class HousingCostsCalculator
 
     delegate :housing_cost_outgoings, to: :@disposable_income_summary
 
-    def initialize(disposable_income_summary:, gross_income_summary:, submission_date:, person:)
+    def initialize(disposable_income_summary:, gross_income_summary:, submission_date:, person:, allow_negative_net: false)
```

Comment:
This doesn't need defaulting - it is supplied by all callers (except tests, which can be fixed)